src/nvim/testdir/test_alot.vim

```diff
@@ -12,6 +12,7 @@ source test_filter_cmd.vim
 source test_filter_map.vim
 source test_float_func.vim
 source test_functions.vim
+source test_ga.vim
```

Comment:
should also be added to https://github.com/ckelsel/neovim/blob/1295e460959a1c463e42a3da1d2c8c4b6746de65/src/nvim/testdir/Makefile#L45 

That's at least necessary for running single tests via `TEST_FILE=foo.res`.